<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Marginal Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.35.4&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<p>
<table border="0" cellspacing="1", cellpadding="3">
<tr><th colspan="12" align="left" class="read_alignment_header_row">Marginal read alignment evidence (highest frequency 20 of 24 shown, sorted by frequency from high to low)</th></tr>
<tr>
<th>&nbsp;</th>
<th>seq&nbsp;id</th>
<th colspan="2">position</th>
<th>ref</th>
<th>new</th>
<th>freq</th>
<th>score&nbsp;(cons/poly)</th>
<th>reads</th>
<th>annotation</th>
<th>genes</th>
<th width="100%">product</th>
</tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_968.html">*</a></td><td align="center">NC_000913</td><td align="right">1,286,965</td><td align="right">0</td><td align="center">A</td><td align="center">G</td><td align="right">63.9%
</td><td align="center">25.7
/&nbsp;45.4
</td><td align="center">66</td><td align="center"><font class="snp_type_nonsynonymous">D86G</font>&nbsp;(G<font class="mutation_in_codon">A</font>C&rarr;G<font class="mutation_in_codon">G</font>C)&nbsp;</td><td align="center"><i>ychS</i></td><td align="left">putative uncharacterized protein YchS</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1018.html">*</a></td><td align="center">NC_000913</td><td align="right">1,636,442</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">50.0%
</td><td align="center">10.1
/&nbsp;10.7
</td><td align="center">18</td><td align="center">intergenic&nbsp;(&#8209;75/&#8209;314)</td><td align="center"><i>nohA/ynfO</i></td><td align="left">Qin prophage; putative prophage DNA&#8209;packaging protein NohA/Qin prophage; protein YnfO</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_854.html">*</a></td><td align="center">NC_000913</td><td align="right">566,205</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">48.6%
</td><td align="center">&#8209;0.8
/&nbsp;132.3
</td><td align="center">108</td><td align="center">pseudogene&nbsp;(157/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_855.html">*</a></td><td align="center">NC_000913</td><td align="right">566,245</td><td align="right">0</td><td align="center">G</td><td align="center">A</td><td align="right">45.6%
</td><td align="center">25.5
/&nbsp;97.4
</td><td align="center">90</td><td align="center">pseudogene&nbsp;(117/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_853.html">*</a></td><td align="center">NC_000913</td><td align="right">566,173</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">44.8%
</td><td align="center">57.1
/&nbsp;159.5
</td><td align="center">105</td><td align="center">pseudogene&nbsp;(189/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1388.html">*</a></td><td align="center">NC_000913</td><td align="right">3,946,791</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">38.5%
</td><td align="center">46.1
/&nbsp;43.8
</td><td align="center">52</td><td align="center">noncoding&nbsp;(92/120&nbsp;nt)</td><td align="center"><i>rrfC</i></td><td align="left">5S ribosomal RNA</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_856.html">*</a></td><td align="center">NC_000913</td><td align="right">566,277</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">38.0%
</td><td align="center">60.7
/&nbsp;67.0
</td><td align="center">80</td><td align="center">pseudogene&nbsp;(85/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1159.html">*</a></td><td align="center">NC_000913</td><td align="right">2,470,427</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">32.7%
</td><td align="center">158.0
/&nbsp;15.7
</td><td align="center">101</td><td align="center"><font class="snp_type_nonsynonymous">Q433K</font>&nbsp;(<font class="mutation_in_codon">C</font>AA&rarr;<font class="mutation_in_codon">A</font>AA)&nbsp;</td><td align="center"><i>yfdI</i></td><td align="left">CPS&#8209;53 (KpLE1) prophage; serotype specific glucosyl transferase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_877.html">*</a></td><td align="center">NC_000913</td><td align="right">708,004</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">27.9%
</td><td align="center">8640.6
/&nbsp;inf
</td><td align="center">5007</td><td align="center">intergenic&nbsp;(+247/&#8209;330)</td><td align="center"><i>glnS/chiP</i></td><td align="left">glutamine&#8209;&#8209;tRNA ligase/chitobiose outer membrane channel</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1108.html">*</a></td><td align="center">NC_000913</td><td align="right">2,074,681</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">25.5%
</td><td align="center">68.6
/&nbsp;15.5
</td><td align="center">49</td><td align="center">intergenic&nbsp;(+23/&#8209;98)</td><td align="center"><i>flu/yeeR</i></td><td align="left">CP4&#8209;44 prophage; self recognizing antigen 43 (Ag43) autotransporter/CP4&#8209;44 prophage; inner membrane protein YeeR</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_876.html">*</a></td><td align="center">NC_000913</td><td align="right">707,976</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">24.0%
</td><td align="center">10115.4
/&nbsp;inf
</td><td align="center">7167</td><td align="center">intergenic&nbsp;(+219/&#8209;358)</td><td align="center"><i>glnS/chiP</i></td><td align="left">glutamine&#8209;&#8209;tRNA ligase/chitobiose outer membrane channel</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1157.html">*</a></td><td align="center">NC_000913</td><td align="right">2,463,552</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">23.9%
</td><td align="center">175.1
/&nbsp;24.5
</td><td align="center">88</td><td align="center"><font class="snp_type_nonsynonymous">G181R</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">C</font>GC)&nbsp;</td><td align="center"><i>yfdF</i></td><td align="left">protein YfdF</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1465.html">*</a></td><td align="center">NC_000913</td><td align="right">4,518,992</td><td align="right">0</td><td align="center">C</td><td align="center">.</td><td align="right">23.0%
</td><td align="center">311.7
/&nbsp;99.0
</td><td align="center">127</td><td align="center">intergenic&nbsp;(+190/+346)</td><td align="center"><i>insA&#8209;7/yjhU</i></td><td align="left">KpLE2 phage&#8209;like element; IS1 repressor protein InsA/KpLE2 phage&#8209;like element; putative DNA&#8209;binding transcriptional regulator YjhU</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1462.html">*</a></td><td align="center">NC_000913</td><td align="right">4,518,958</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">22.7%
</td><td align="center">272.5
/&nbsp;58.7
</td><td align="center">110</td><td align="center">intergenic&nbsp;(+156/+380)</td><td align="center"><i>insA&#8209;7/yjhU</i></td><td align="left">KpLE2 phage&#8209;like element; IS1 repressor protein InsA/KpLE2 phage&#8209;like element; putative DNA&#8209;binding transcriptional regulator YjhU</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_798.html">*</a></td><td align="center">NC_000913</td><td align="right">357,775</td><td align="right">0</td><td align="center">A</td><td align="center">T</td><td align="right">22.6%
</td><td align="center">102.1
/&nbsp;28.7
</td><td align="center">53</td><td align="center">intergenic&nbsp;(+321/+16)</td><td align="center"><i>codA/cynR</i></td><td align="left">cytosine/isoguanine deaminase/DNA&#8209;binding transcriptional dual regulator CynR</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1422.html">*</a></td><td align="center">NC_000913</td><td align="right">4,296,060</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">22.6%
</td><td align="center">192.9
/&nbsp;44.8
</td><td align="center">102</td><td align="center">intergenic&nbsp;(+266/+376)</td><td align="center"><i>gltP/yjcO</i></td><td align="left">glutamate/aspartate : H(+) symporter GltP/Sel1 repeat&#8209;containing protein YjcO</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1463.html">*</a></td><td align="center">NC_000913</td><td align="right">4,518,963</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">22.1%
</td><td align="center">282.3
/&nbsp;77.5
</td><td align="center">113</td><td align="center">intergenic&nbsp;(+161/+375)</td><td align="center"><i>insA&#8209;7/yjhU</i></td><td align="left">KpLE2 phage&#8209;like element; IS1 repressor protein InsA/KpLE2 phage&#8209;like element; putative DNA&#8209;binding transcriptional regulator YjhU</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1250.html">*</a></td><td align="center">NC_000913</td><td align="right">3,013,482</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">22.0%
</td><td align="center">99.0
/&nbsp;14.8
</td><td align="center">59</td><td align="center"><font class="snp_type_nonsynonymous">N253T</font>&nbsp;(A<font class="mutation_in_codon">A</font>C&rarr;A<font class="mutation_in_codon">C</font>C)&nbsp;</td><td align="center"><i>yqeB</i></td><td align="left">XdhC&#8209;CoxI family protein YqeB</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1464.html">*</a></td><td align="center">NC_000913</td><td align="right">4,518,966</td><td align="right">0</td><td align="center">A</td><td align="center">G</td><td align="right">21.6%
</td><td align="center">234.6
/&nbsp;50.3
</td><td align="center">116</td><td align="center">intergenic&nbsp;(+164/+372)</td><td align="center"><i>insA&#8209;7/yjhU</i></td><td align="left">KpLE2 phage&#8209;like element; IS1 repressor protein InsA/KpLE2 phage&#8209;like element; putative DNA&#8209;binding transcriptional regulator YjhU</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_799.html">*</a></td><td align="center">NC_000913</td><td align="right">357,778</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">21.4%
</td><td align="center">101.5
/&nbsp;20.5
</td><td align="center">56</td><td align="center">intergenic&nbsp;(+324/+13)</td><td align="center"><i>codA/cynR</i></td><td align="left">cytosine/isoguanine deaminase/DNA&#8209;binding transcriptional dual regulator CynR</td></tr>
</table>

<p>
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="12" align="left" class="new_junction_header_row">Marginal new junction evidence (sorted from low to high skew)</th></tr>
<!-- Header Lines for New Junction -->
<tr>
<th colspan="2">&nbsp;</th>
<th>seq&nbsp;id</th>
<th>position</th>
<th>reads&nbsp;(cov)</th>
<th>reads&nbsp;(cov)</th>
<th>score</th>
<th>skew</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">product</th>
</tr>

<!-- Item Lines for New Junction -->
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_1505.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_1505.html">?</a></td>
<td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">70972&nbsp;=</td><td align="center" class="junction_gene">13830 (176.980)</td><td rowspan="2" align="center">336 (4.370)<br>+GG</td>
<td rowspan="2" align="center">15/262</td>
<td rowspan="2" align="center">1.9</td>
<td rowspan="2" align="center">3.2%</td>
<td align="center" class="junction_gene">coding&nbsp;(586/879&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>araC</i></td>
<td class="junction_gene">DNA&#8209;binding transcriptional dual regulator AraC</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_1505.html">?</a></td><td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">=&nbsp;71213</td>
<td align="center" class="junction_gene">6847 (87.620)</td><td align="center" class="junction_gene">coding&nbsp;(827/879&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>araC</i></td>
<td class="junction_gene">DNA&#8209;binding transcriptional dual regulator AraC</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_1504.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_1504.html">?</a></td>
<td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">=&nbsp;70904</td><td align="center" class="junction_gene">13477 (172.470)</td><td rowspan="2" align="center">6 (0.080)</td>
<td rowspan="2" align="center">5/262</td>
<td rowspan="2" align="center">3.8</td>
<td rowspan="2" align="center">0.045%</td>
<td align="center" class="junction_gene">coding&nbsp;(518/879&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>araC</i></td>
<td class="junction_gene">DNA&#8209;binding transcriptional dual regulator AraC</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_1504.html">?</a></td><td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">=&nbsp;70919</td>
<td align="center" class="junction_gene">13152 (170.900)</td><td align="center" class="junction_gene">coding&nbsp;(533/879&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>araC</i></td>
<td class="junction_gene">DNA&#8209;binding transcriptional dual regulator AraC</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_1503.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_1503.html">?</a></td>
<td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">70646&nbsp;=</td><td align="center" class="junction_gene">12316 (157.610)</td><td rowspan="2" align="center">6 (0.080)</td>
<td rowspan="2" align="center">5/258</td>
<td rowspan="2" align="center">3.8</td>
<td rowspan="2" align="center">0.053%</td>
<td align="center" class="junction_gene">coding&nbsp;(260/879&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>araC</i></td>
<td class="junction_gene">DNA&#8209;binding transcriptional dual regulator AraC</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_1503.html">?</a></td><td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">70694&nbsp;=</td>
<td align="center" class="junction_gene">10701 (141.220)</td><td align="center" class="junction_gene">coding&nbsp;(308/879&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>araC</i></td>
<td class="junction_gene">DNA&#8209;binding transcriptional dual regulator AraC</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_1521.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_1521.html">?</a></td>
<td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">=&nbsp;2902341</td><td align="center" class="junction_gene">91 (1.210)</td><td rowspan="2" align="center">5 (0.070)<br>+GGT</td>
<td rowspan="2" align="center">3/250</td>
<td rowspan="2" align="center">4.4</td>
<td rowspan="2" align="center">5.2%</td>
<td align="center" class="junction_gene">coding&nbsp;(446/1479&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ygcE</i></td>
<td class="junction_gene">putative sugar kinase YgcE</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_1521.html">?</a></td><td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">2902342&nbsp;=</td>
<td align="center" class="junction_gene">92 (1.180)</td><td align="center" class="junction_gene">coding&nbsp;(447/1479&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ygcE</i></td>
<td class="junction_gene">putative sugar kinase YgcE</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_1518.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_1518.html">?</a></td>
<td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">2433854&nbsp;=</td><td align="center" class="junction_gene">82 (1.050)</td><td rowspan="2" align="center">3 (0.040)</td>
<td rowspan="2" align="center">3/260</td>
<td rowspan="2" align="center">4.5</td>
<td rowspan="2" align="center">3.9%</td>
<td align="center" class="junction_gene">coding&nbsp;(73/915&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>accD</i></td>
<td class="junction_gene">acetyl&#8209;CoA carboxyltransferase subunit beta</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_1518.html">?</a></td><td rowspan="1" class="junction_gene">NC_000913</td>
<td align="center" class="junction_gene">2433892&nbsp;=</td>
<td align="center" class="junction_gene">67 (0.880)</td><td align="center" class="junction_gene">coding&nbsp;(35/915&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>accD</i></td>
<td class="junction_gene">acetyl&#8209;CoA carboxyltransferase subunit beta</td>
</tr>
</table>
</body></html>