<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Marginal Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.35.4&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<p>
<table border="0" cellspacing="1", cellpadding="3">
<tr><th colspan="12" align="left" class="read_alignment_header_row">Marginal read alignment evidence (highest frequency 20 of 22 shown, sorted by frequency from high to low)</th></tr>
<tr>
<th>&nbsp;</th>
<th>seq&nbsp;id</th>
<th colspan="2">position</th>
<th>ref</th>
<th>new</th>
<th>freq</th>
<th>score&nbsp;(cons/poly)</th>
<th>reads</th>
<th>annotation</th>
<th>genes</th>
<th width="100%">product</th>
</tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1385.html">*</a></td><td align="center">NC_000913</td><td align="right">3,946,791</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">60.0%
</td><td align="center">&#8209;0.1
/&nbsp;31.8
</td><td align="center">35</td><td align="center">noncoding&nbsp;(92/120&nbsp;nt)</td><td align="center"><i>rrfC</i></td><td align="left">5S ribosomal RNA</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_967.html">*</a></td><td align="center">NC_000913</td><td align="right">1,286,965</td><td align="right">0</td><td align="center">A</td><td align="center">G</td><td align="right">53.6%
</td><td align="center">&#8209;0.1
/&nbsp;58.8
</td><td align="center">61</td><td align="center"><font class="snp_type_nonsynonymous">D86G</font>&nbsp;(G<font class="mutation_in_codon">A</font>C&rarr;G<font class="mutation_in_codon">G</font>C)&nbsp;</td><td align="center"><i>ychS</i></td><td align="left">putative uncharacterized protein YchS</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_858.html">*</a></td><td align="center">NC_000913</td><td align="right">566,205</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">45.6%
</td><td align="center">8.3
/&nbsp;78.0
</td><td align="center">68</td><td align="center">pseudogene&nbsp;(157/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_859.html">*</a></td><td align="center">NC_000913</td><td align="right">566,245</td><td align="right">0</td><td align="center">G</td><td align="center">A</td><td align="right">44.1%
</td><td align="center">22.2
/&nbsp;60.0
</td><td align="center">59</td><td align="center">pseudogene&nbsp;(117/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_857.html">*</a></td><td align="center">NC_000913</td><td align="right">566,173</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">37.7%
</td><td align="center">64.8
/&nbsp;92.7
</td><td align="center">69</td><td align="center">pseudogene&nbsp;(189/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_860.html">*</a></td><td align="center">NC_000913</td><td align="right">566,277</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">31.6%
</td><td align="center">66.4
/&nbsp;37.9
</td><td align="center">57</td><td align="center">pseudogene&nbsp;(85/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_802.html">*</a></td><td align="center">NC_000913</td><td align="right">357,778</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">30.9%
</td><td align="center">58.0
/&nbsp;25.4
</td><td align="center">42</td><td align="center">intergenic&nbsp;(+324/+13)</td><td align="center"><i>codA/cynR</i></td><td align="left">cytosine/isoguanine deaminase/DNA&#8209;binding transcriptional dual regulator CynR</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_801.html">*</a></td><td align="center">NC_000913</td><td align="right">357,775</td><td align="right">0</td><td align="center">A</td><td align="center">T</td><td align="right">30.2%
</td><td align="center">66.1
/&nbsp;33.9
</td><td align="center">43</td><td align="center">intergenic&nbsp;(+321/+16)</td><td align="center"><i>codA/cynR</i></td><td align="left">cytosine/isoguanine deaminase/DNA&#8209;binding transcriptional dual regulator CynR</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_884.html">*</a></td><td align="center">NC_000913</td><td align="right">708,004</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">28.2%
</td><td align="center">5107.2
/&nbsp;inf
</td><td align="center">3012</td><td align="center">intergenic&nbsp;(+247/&#8209;330)</td><td align="center"><i>glnS/chiP</i></td><td align="left">glutamine&#8209;&#8209;tRNA ligase/chitobiose outer membrane channel</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1153.html">*</a></td><td align="center">NC_000913</td><td align="right">2,463,552</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">27.5%
</td><td align="center">153.8
/&nbsp;23.8
</td><td align="center">80</td><td align="center"><font class="snp_type_nonsynonymous">G181R</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">C</font>GC)&nbsp;</td><td align="center"><i>yfdF</i></td><td align="left">protein YfdF</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1459.html">*</a></td><td align="center">NC_000913</td><td align="right">4,518,963</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">26.2%
</td><td align="center">247.8
/&nbsp;84.1
</td><td align="center">103</td><td align="center">intergenic&nbsp;(+161/+375)</td><td align="center"><i>insA&#8209;7/yjhU</i></td><td align="left">KpLE2 phage&#8209;like element; IS1 repressor protein InsA/KpLE2 phage&#8209;like element; putative DNA&#8209;binding transcriptional regulator YjhU</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1155.html">*</a></td><td align="center">NC_000913</td><td align="right">2,470,427</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">25.4%
</td><td align="center">223.9
/&nbsp;11.0
</td><td align="center">111</td><td align="center"><font class="snp_type_nonsynonymous">Q433K</font>&nbsp;(<font class="mutation_in_codon">C</font>AA&rarr;<font class="mutation_in_codon">A</font>AA)&nbsp;</td><td align="center"><i>yfdI</i></td><td align="left">CPS&#8209;53 (KpLE1) prophage; serotype specific glucosyl transferase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1458.html">*</a></td><td align="center">NC_000913</td><td align="right">4,518,958</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">24.5%
</td><td align="center">250.1
/&nbsp;67.5
</td><td align="center">102</td><td align="center">intergenic&nbsp;(+156/+380)</td><td align="center"><i>insA&#8209;7/yjhU</i></td><td align="left">KpLE2 phage&#8209;like element; IS1 repressor protein InsA/KpLE2 phage&#8209;like element; putative DNA&#8209;binding transcriptional regulator YjhU</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1460.html">*</a></td><td align="center">NC_000913</td><td align="right">4,518,966</td><td align="right">0</td><td align="center">A</td><td align="center">G</td><td align="right">24.0%
</td><td align="center">199.5
/&nbsp;54.6
</td><td align="center">105</td><td align="center">intergenic&nbsp;(+164/+372)</td><td align="center"><i>insA&#8209;7/yjhU</i></td><td align="left">KpLE2 phage&#8209;like element; IS1 repressor protein InsA/KpLE2 phage&#8209;like element; putative DNA&#8209;binding transcriptional regulator YjhU</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1461.html">*</a></td><td align="center">NC_000913</td><td align="right">4,518,992</td><td align="right">0</td><td align="center">C</td><td align="center">.</td><td align="right">23.6%
</td><td align="center">274.2
/&nbsp;88.6
</td><td align="center">110</td><td align="center">intergenic&nbsp;(+190/+346)</td><td align="center"><i>insA&#8209;7/yjhU</i></td><td align="left">KpLE2 phage&#8209;like element; IS1 repressor protein InsA/KpLE2 phage&#8209;like element; putative DNA&#8209;binding transcriptional regulator YjhU</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_883.html">*</a></td><td align="center">NC_000913</td><td align="right">707,976</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">23.4%
</td><td align="center">5951.9
/&nbsp;inf
</td><td align="center">4190</td><td align="center">intergenic&nbsp;(+219/&#8209;358)</td><td align="center"><i>glnS/chiP</i></td><td align="left">glutamine&#8209;&#8209;tRNA ligase/chitobiose outer membrane channel</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1344.html">*</a></td><td align="center">NC_000913</td><td align="right">3,712,935</td><td align="right">0</td><td align="center">T</td><td align="center">A</td><td align="right">22.9%
</td><td align="center">260.6
/&nbsp;32.3
</td><td align="center">131</td><td align="center">intergenic&nbsp;(&#8209;1/&#8209;157)</td><td align="center"><i>yhjY/tag</i></td><td align="left">conserved protein YhjY/3&#8209;methyl&#8209;adenine DNA glycosylase I, constitutive</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_1345.html">*</a></td><td align="center">NC_000913</td><td align="right">3,712,937</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">22.0%
</td><td align="center">207.2
/&nbsp;27.8
</td><td align="center">118</td><td align="center">intergenic&nbsp;(&#8209;3/&#8209;155)</td><td align="center"><i>yhjY/tag</i></td><td align="left">conserved protein YhjY/3&#8209;methyl&#8209;adenine DNA glycosylase I, constitutive</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_861.html">*</a></td><td align="center">NC_000913</td><td align="right">566,323</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">21.1%
</td><td align="center">92.9
/&nbsp;18.7
</td><td align="center">52</td><td align="center">pseudogene&nbsp;(39/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_862.html">*</a></td><td align="center">NC_000913</td><td align="right">566,326</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">21.1%
</td><td align="center">85.8
/&nbsp;20.5
</td><td align="center">52</td><td align="center">pseudogene&nbsp;(36/264&nbsp;nt)</td><td align="center"><i>exoD</i></td><td align="left">DLP12 prophage; putative exonuclease ExoD</td></tr>
</table>

<p>
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="12" align="left" class="new_junction_header_row">Marginal new junction evidence (sorted from low to high skew)</th></tr>
<!-- Header Lines for New Junction -->
<tr>
<th colspan="2">&nbsp;</th>
<th>seq&nbsp;id</th>
<th>position</th>
<th>reads&nbsp;(cov)</th>
<th>reads&nbsp;(cov)</th>
<th>score</th>
<th>skew</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">product</th>
</tr>

<!-- Item Lines for New Junction -->
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_1501.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_1501.html">?</a></td>
<td rowspan="1" class="junction_repeat">NC_000913</td>
<td align="center" class="junction_repeat">381292&nbsp;=</td><td align="center" class="junction_repeat">NA (NA)</td><td rowspan="2" align="center">3 (0.050)</td>
<td rowspan="2" align="center">3/250</td>
<td rowspan="2" align="center">4.0</td>
<td rowspan="2" align="center">NA</td>
<td align="center" class="junction_repeat">noncoding&nbsp;(33/1331&nbsp;nt)</td>
<td align="center" class="junction_repeat"><i>IS2</i></td>
<td class="junction_repeat">repeat region</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_1501.html">?</a></td><td rowspan="1" class="junction_repeat">NC_000913</td>
<td align="center" class="junction_repeat">381312&nbsp;=</td>
<td align="center" class="junction_repeat">NA (NA)</td><td align="center" class="junction_repeat">noncoding&nbsp;(53/1331&nbsp;nt)</td>
<td align="center" class="junction_repeat"><i>IS2</i></td>
<td class="junction_repeat">repeat region</td>
</tr>
</table>
</body></html>